src/Codeception/Util/Debug.php

```diff
@@ -38,6 +40,35 @@ public static function isEnabled(): bool
         return (bool)self::$output;
     }
 
+    public static function pause(array $vars = []): void
+    {
+        if (!self::isEnabled()) {
+            return;
+        }
+
+        $pauseShell = new PauseShell();
+        $psy = $pauseShell->getShell();
+        $psy->setScopeVariables($vars);
+        $backtrace = debug_backtrace(DEBUG_BACKTRACE_PROVIDE_OBJECT, 3);
+        foreach ($backtrace as $backtraceStep) {
+            $class = $backtraceStep['class'] ?? null;
+            $fn = $backtraceStep['class'] ?? null;
```

Comment:
This line should be using `$backtraceStep['function']`, right?

Reply:
Ups, right!